homeassistant/components/input_number/__init__.py

```diff
@@ -211,52 +212,53 @@ def from_yaml(cls, config: dict) -> InputNumber:
         return input_num
 
     @property
-    def should_poll(self):
+    def should_poll(self) -> bool:
         """If entity should be polled."""
         return False
 
     @property
     def _minimum(self) -> float:
         """Return minimum allowed value."""
-        return self._config[CONF_MIN]
+        """convert to float"""
+        return float(self._config[CONF_MIN])
```

Comment:
Already done by the schema.

Reply:
I've tried to avoid that, but without casting, I get this error:

> homeassistant/components/input_number/__init__.py:222: error: Returning Any from function declared to return "float"  [no-any-return]

I'll take a look at TypedDict and usages, maybe that will give me a hint

Reply:
The idea is that you declare a TypedDict which has defined keys according to what will be passed to the `InputNumber` init function by the storage helper.